app/models.py

```diff
@@ -38,6 +38,10 @@ class Lot(db.Model):
     name = db.Column(db.String, nullable=False)
     one_service_limit = db.Column(db.Boolean, nullable=False, default=False)
 
+    @property
+    def requires_brief(self):
```

Comment:
The flag controls whether it's possible to create a brief for a given lot, so should it be `allows_briefs` rather than "requires"?